packages/monaco/src/browser/monaco-command.ts

```diff
@@ -120,6 +126,77 @@ export class MonacoEditorCommandHandlers implements CommandContribution {
             }
         };
     }
+    protected newConfigIndentationHandler(): MonacoEditorCommandHandler {
+        return {
+            execute: (editor: MonacoEditor) => {
+                const configTabSize = (useSpaces: boolean) => {
+                    const editorModel = editor.document;
+                    if (editorModel && editorModel.textEditorModel) {
+                        const tabSize = editorModel.textEditorModel.getOptions().tabSize;
+                        const tabSizeOptions = Array.from(Array(8), (_, x) => x + 1)
+                            .map(size => new UpdateTabSizeOpenItem(size === tabSize ? `${size}   Configured Tab Size` : size.toString(),
```

Comment:
I've noticed that `Configure Tab Size` is always against `4` in VS code and the current size is selected by default. You should be able to achieve it with https://github.com/theia-ide/theia/blob/c145b1533c69fb2e560099b85bc5b12c18bd94b8/packages/core/src/browser/quick-open/quick-open-service.ts#L27